src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/Cas1ApAreaEmailAddressSeedJob.kt

```diff
@@ -32,8 +32,6 @@ class Cas1ApAreaEmailAddressSeedJob(
       error("Email address for '${row.apAreaIdentifier}' is blank")
     }
 
-    apAreaRepository.updateEmailAddress(apArea.id, emailAddress)
-
```

Comment:
i think this whole seed job can be removed as its only purpose was to update the email address. we have an equivalent for cru manage areas already. Could you please remove this, and remove the `approvedPremisesApAreaEmailAddresses` enumeration in the API used by the SeedService?

Reply:
Have removed this job now.